apps/site/src/views/Home/components/ApplyButton/ApplyButton.tsx

```diff
@@ -1,17 +1,38 @@
 import Button from "react-bootstrap/Button";
+import StickerPosition from "@/components/Sticker/StickerPosition";
+import { HackSticker, HeartSticker } from "@/components/Sticker/Stickers";
 
 import styles from "./ApplyButton.module.scss";
 
 export default function ApplyButton() {
 	return (
-		<Button
-			className={styles.applyButton}
-			href="/apply"
-			variant=""
-			target="_blank"
-			disabled
+		<StickerPosition
+			stickers={[
+				{
+					Node: HeartSticker,
+					positionX: "right",
+					positionY: "bottom",
+					offsetX: 50,
+					offsetY: 50,
+				},
+				{
+					Node: HackSticker,
```

Comment:
Could you make the Hack sticker a bit smaller? Or find a way to make the size of the sticker responsive? I feel it looks a bit big on mobile views and it covers the title text. I would potentially suggest to address in another PR, but we don't have a staging environment, and this is on our home page.

Reply:
I agree with Tyler here. It also seems to cover the title a little, at least on my screen.

Reply:
I just included the Hack sticker as a demo for how the component works. I'll remove it and we just merge this PR for now? @tyleryy?

Reply:
yea sg